src/sdk/auth.js

```diff
@@ -156,6 +161,19 @@ export class OCAuthCore
         }
         return {};
     }
+
+    get ocId()
```

Comment:
hey I think we can go with this direction.
but
```
get OCId() {
    return this.authInfoManager.getAuthInfo().edu_username;
}

get ethAddress() {
 .... similar ...
}
```

Reply:
I think in addition to this, we can still retain what we did in the React components. 
- my previous comment was that if we *only* have it in react component, then it doesn't serve the higher goal. 
- But we may also add that into react component so people can access it perhaps with the hook. So we can also include essential info such as OCId and ethAddress in the context directly.